app/api/speakers_calls.py

```diff
@@ -89,14 +95,20 @@ def before_patch(self, args, kwargs, data):
                     event_id=kwargs['event_id']
                 ).one()
                 event = speakers_call.event
-                if (
-                    speakers_call.starts_at > event.starts_at
-                    or speakers_call.ends_at > event.starts_at
-                ):
-                    raise ForbiddenError(
-                        {'source': ''},
-                        "Speakers call date can\'t be after the event start date",
-                    )
+                if event.is_cfs_enabled:
+                    if (
```

Comment:
Should be checked nevertheless

Reply:
means that event if cfs is not enabled, this check should be there ? 
```
           if (
                    speakers_call.starts_at > event.starts_at
                    or speakers_call.ends_at > event.starts_at
                ):
                    raise ForbiddenError(
                        {'source': ''},
                        "Speakers call date can\'t be after the event start date",
                    )
```

Reply:
Yes